core-text/src/font.rs

```diff
@@ -660,3 +661,13 @@ fn copy_font() {
     assert_eq!(font.family_name(), "Zapf Dingbats");
 }
 
+#[test]
+fn copy_system_font() {
+    let small = unsafe {
+        CTFont::wrap_under_create_rule(
+        CTFontCreateUIFontForLanguage(kCTFontSystemDetailFontType, 19., std::ptr::null())
```

Comment:
nit: indentation.

Reply:
> ```
> thread 'font::copy_system_font' panicked at 'assertion failed: `(left == right)`
> 
>   left: `Some(file:///System/Library/Fonts/SFNSDisplay-Regular.otf)`,
> 
>  right: `Some(file:///System/Library/Fonts/SFNSText-Regular.otf)`', core-text/src/font.rs:672:5
> 
> note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
> 
> test font::copy_system_font ... FAILED
> ```

This was intentional. I plan to use these tests to help document the semantics of these functions.